### DISCUSSION TITLE
Ajuda com https://github.com/nfephp-org/sped-da/

### DISCUSSION THREAD

**Discussion by micheleaparecidat1** [community/community#127392] [Programming Help]
### Body

Olá bom dia!!!
Tudo bem? 
Gostaria de saber se alguém teria o modelo em pdf da etiqueta do danfe ([danfeetiqueta.php](https://github.com/nfephp-org/sped-da/blob/master/examples/nfe/danfeetiqueta.php) desse projeto implementada pra me encaminhar por favor? 
Obrigada!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @micheleaparecidat1, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
